dev-app/src/main/java/com/uid2/dev/DevApplication.kt

```diff
@@ -15,6 +20,12 @@ class DevApplication : Application() {
         // Alternatively, we could initialise the UID2Manager with our own custom NetworkSession...
         // UID2Manager.init(this, INTEG_SERVER_URL, OkNetworkSession(), true)
 
+        // Create the Prebid integration and allow it to start observing the UID2Manager instance.
+        PrebidMobile.initializeSdk(this) { Log.i(TAG, "Prebid: $it") }
+        prebid = UID2Prebid(UID2Manager.getInstance()).apply {
```

Comment:
```suggestion
        prebid = UID2Prebid().apply {
```

you probably don't need this anymore